vagrant/ansible/roles/samba-glusterfs.setup/files/smb.conf

```diff
@@ -0,0 +1,20 @@
+[global]
+passdb backend = tdbsam
+aio read size = 4096
+
+log level = 1
+host msdfs = yes
+msdfs root = yes
+msdfs shuffle referrals = yes
+
+ea support = yes
```

Comment:
`ea support` is enabled by default. we can remove it

Reply:
done